assets/stylesheets/_components.scss

```diff
@@ -175,6 +175,7 @@
 @import 'me/credit-cards/credit-card-delete';
 @import 'me/credit-cards/credit-cards';
 @import 'me/get-apps/style';
+@import 'me/gravatar-updater/style';
```

Comment:
Thanks for the review :) That structure does seem more obvious. I'll re-arrange it.